src/azure-cli/azure/cli/command_modules/monitor/aaz/latest/monitor/log_profiles/_list.py

```diff
@@ -0,0 +1,201 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#
+# Code generated by aaz-dev-tools
+# --------------------------------------------------------------------------------------------
+
+# pylint: skip-file
+# flake8: noqa
+
+from azure.cli.core.aaz import *
+
+
+@register_command(
+    "monitor log-profiles list",
+)
+class List(AAZCommand):
+    """List the log profiles. :keyword callable cls: A custom type or function that will be passed the direct response:return: An iterator like instance of either LogProfileResource or the result of cls(response) :rtype:  ~azure.core.paging.ItemPaged[~$(python-base-namespace).v2016_03_01.models.LogProfileResource] :raises ~azure.core.exceptions.HttpResponseError:.
```

Comment:
It seems the command help has invalid content.

Reply:
origin list help message

Reply:
origin list help message

Reply:
origin list help message